app/views/results/index.html.erb

```diff
@@ -9,13 +9,15 @@
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-one-third">
-      filters go here
       <div data-qa="filters">
+        <%= render partial: 'results/location' %>
         <div>
           <%=link_to "Change location or choose a provider", @results_view.filter_path_with_unescaped_commas(location_path), class: "govuk-link", data: { qa: "filters__location_link" }%>
         </div>
         <%= render partial: 'results/subjects' %>
-
+        <div>
+          <%=link_to "Change subjects", @results_view.filter_path_with_unescaped_commas(subject_path), class: "govuk-link", data: { qa: "filters__subject_link" }%>
```

Comment:
Should this be here?

Reply:
![image](https://user-images.githubusercontent.com/470137/74174830-8f059080-4c2c-11ea-9623-d41c3a679cf9.png)

Reply:
no

Reply:
drop it 
![image](https://user-images.githubusercontent.com/470137/74336957-49190b80-4d97-11ea-8d0a-074f0c4f0eea.png)